src/assets/stylesheets/index.scss

```diff
@@ -617,7 +617,7 @@ $easing: cubic-bezier(0.7, 0, 0, 0.7);
   .h4,
   .h5,
   .h6,
-  .table-features,
+  .table-widget,
```

Comment:
This rings an 🚨 for me. Ideally all styling needed for a widget should be in its corresponding `.scss` file. But this here indicates, that their is some global styling to be also considered.

I think it is better, if we duplicate the `color: #fff` part for `.bg-dark-image` in `TableWidget.scss`.

This also applies to the `.footer` section.